config/fund_loader_config/common_fund_config/fund_base_tree_paths.py

```diff
@@ -1,3 +1,5 @@
+# Should increment for each new round
 COF_R2_W2_BASE_PATH = 1
 COF_R2_W3_BASE_PATH = 1
 COF_R3W1_BASE_PATH = 2
+NIGHT_SHELTER_BASE_PATH = 3
```

Comment:
just a thought, can we use abbreviation for Night shelter( `NS_BASE_PATH` ) just like COF